### ISSUE TITLE
freight-agent: symbol collision with newer xmlrpc

### ISSUE THREAD

**Issue by igor-raits** [FreightAgent/freight-tools#5]
```c
xmlrpc_client.c:35:12: error: conflicting types for 'xmlrpc_init'
 static int xmlrpc_init(struct agent_config *acfg)
            ^~~~~~~~~~~
In file included from /usr/include/xmlrpc-c/client.h:15:0,
                 from xmlrpc_client.c:27:
/usr/include/xmlrpc-c/base.h:45:1: note: previous declaration of 'xmlrpc_init' was here
 xmlrpc_init(xmlrpc_env * const envP);
 ^~~~~~~~~~~
```

Should be easy to just rename I suppose.

**Comment by kkofler** [comment]
It's a static function, so it cannot be referenced by name from outside the file. Thus, simply replacing it with any non-conflicting identifier in the file will fix the build (e.g., `sed -i -e 's/xmlrpc_init/xmlrpc_init_acfg/g' src/freight-agent/xmlrpc_client.c`).

**Comment by nhorman** [comment]
Thanks!  I'll take care of this shortly!

**Comment by nhorman** [comment]
fixed in commit 5c450908c95f93deb6a2f5bcf227ab855f2fdd0e.  Thanks!
